# This message describes a Grasp object as in gpg/src/grasp.h

# grasp frame
geometry_msgs/Pose pose
gpd/FingerHand finger_hand # a finger hand message with hand details